src/main/java/com/sprint/mission/discodeit/service/featureService/AuthServiceImpl.java

```diff
@@ -0,0 +1,25 @@
+package com.sprint.mission.discodeit.service.featureService;
+
+import com.sprint.mission.discodeit.dto.auth.LoginDTO;
+import com.sprint.mission.discodeit.entity.User;
+import com.sprint.mission.discodeit.repository.UserRepository;
+import com.sprint.mission.discodeit.service.AuthService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.Map;
+import java.util.NoSuchElementException;
+import java.util.UUID;
+
+@Service
+@RequiredArgsConstructor
+public class AuthServiceImpl implements AuthService {
+    private final UserRepository userRepository;
+    @Override
+    public User login(LoginDTO loginDTO) {
+        Map<UUID, User> userMap = userRepository.findAll();
```

Comment:
```suggestion
        User user = userRepository.findByUserName(loginDTO.userName());
```
우리가 원하는 사용자만 찾으면 어때요?